tests/materialize/materialize-bigquery/snapshot.json

```diff
@@ -1,521 +1,313 @@
 ]
 [
   10,
   "[1,2]",
   "c2F5xY1uYXJhCg==",
   true,
   1010.101,
-  "2023-07-12T18:29:04.671352Z",
+  "1970-01-01T01:00:23Z",
   "null",
   "{\"id\":\"i10\"}",
   10,
   "str10 v2",
-  "{\"_meta\":{\"uuid\":\"fb0cc8b0-20e1-11ee-990b-ffd12dfcd47f\"},\"array_int\":[1,2],\"binary_field\":\"c2F5xY1uYXJhCg==\",\"bool_field\":true,\"float_field\":1010.101,\"id\":10,\"nested\":{\"id\":\"i10\"},\"nullable_int\":10,\"str_field\":\"str10 v2\"}"
+  "{\"_meta\":{\"uuid\":\"82fb2d80-1dda-11b2-8000-071353030311\"},\"array_int\":[1,2],\"binary_field\":\"c2F5xY1uYXJhCg==\",\"bool_field\":true,\"float_field\":1010.101,\"id\":10,\"nested\":{\"id\":\"i10\"},\"nullable_int\":10,\"str_field\":\"str10 v2\"}"
 ]
 [
   1,
   "0001-01-01",
   "0001-01-01T00:00:00Z",
-  "2023-07-12T18:27:01.912219Z",
+  "1970-01-01T01:00:13Z",
   "1",
   "10",
   1.1,
   10.1,
   "00:00:00Z",
-  "{\"_meta\":{\"uuid\":\"b1e13a0e-20e1-11ee-990b-ffd12dfcd47f\"},\"date\":\"0000-01-01\",\"datetime\":\"0000-01-01T00:00:00Z\",\"id\":1,\"int_and_str\":1,\"int_str\":\"10\",\"num_and_str\":1.1,\"num_str\":\"10.1\",\"time\":\"00:00:00Z\"}"
+  "{\"_meta\":{\"uuid\":\"7d054c80-1dda-11b2-8000-071353030311\"},\"date\":\"0000-01-01\",\"datetime\":\"0000-01-01T00:00:00Z\",\"id\":1,\"int_and_str\":1,\"int_str\":\"10\",\"num_and_str\":1.1,\"num_str\":\"10.1\",\"time\":\"00:00:00Z\"}"
 ]
 [
   2,
   "1999-02-02",
   "1999-02-02T14:20:12.33Z",
-  "2023-07-12T18:27:12.35461Z",
+  "1970-01-01T01:00:14Z",
   "2",
   "20",
   2.1,
   20.1,
   "14:20:12.33Z",
-  "{\"_meta\":{\"uuid\":\"b81a9bf4-20e1-11ee-990b-ffd12dfcd47f\"},\"date\":\"1999-02-02\",\"datetime\":\"1999-02-02T14:20:12.33Z\",\"id\":2,\"int_and_str\":2,\"int_str\":\"20\",\"num_and_str\":2.1,\"num_str\":\"20.1\",\"time\":\"14:20:12.33Z\"}"
+  "{\"_meta\":{\"uuid\":\"7d9de300-1dda-11b2-8000-071353030311\"},\"date\":\"1999-02-02\",\"datetime\":\"1999-02-02T14:20:12.33Z\",\"id\":2,\"int_and_str\":2,\"int_str\":\"20\",\"num_and_str\":2.1,\"num_str\":\"20.1\",\"time\":\"14:20:12.33Z\"}"
 ]
 [
   3,
   "1000-03-03",
   "1000-03-03T23:59:38.099999Z",
-  "2023-07-12T18:18:11.537199Z",
+  "1970-01-01T00:00:11Z",
   "3",
   "30",
   3.1,
   30.1,
   "23:59:38.10Z",
-  "{\"_meta\":{\"uuid\":\"75c06bd6-20e0-11ee-990b-ffd12dfcd47f\"},\"date\":\"1000-03-03\",\"datetime\":\"1000-03-03T23:59:38.10Z\",\"id\":3,\"int_and_str\":3,\"int_str\":\"30\",\"num_and_str\":3.1,\"num_str\":\"30.1\",\"time\":\"23:59:38.10Z\"}"
+  "{\"_meta\":{\"uuid\":\"1a0fb780-1dd2-11b2-8000-071353030311\"},\"date\":\"1000-03-03\",\"datetime\":\"1000-03-03T23:59:38.10Z\",\"id\":3,\"int_and_str\":3,\"int_str\":\"30\",\"num_and_str\":3.1,\"num_str\":\"30.1\",\"time\":\"23:59:38.10Z\"}"
 ]
 [
   4,
   "2023-08-29",
   "2023-08-29T23:59:38Z",
-  "2023-07-12T18:18:48.441281Z",
+  "1970-01-01T00:00:12Z",
```

Comment:
These dates seem kind of weird...is it an artifact of using flowctl preview?

Reply:
@williamhbaker yeah, we previously would provide hardcoded `_meta/uuid`s as part of the fixture, but now flowctl preview takes care of generating those.